test/integration/rhel-8.0/gcc-static-local-var-4.test

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 
-if $(nm test-gcc-static-local-var-4.ko | grep -q free_ioctx); then
```

Comment:
I assume this should be renamed to .LOADED as well.

Reply:
Oops, I'll go rename that one, too